src/pages/SignIn/components/GoogleSignInButton.jsx

```diff
@@ -1,14 +1,16 @@
 import { GoogleAuthProvider, signInWithPopup } from 'firebase/auth';
 import { auth } from '../../../utils/firebase';
 import { Button } from '@nextui-org/react';
+import Google from '../../../assets/google.svg';
 
 const GoogleSignInButton = () => {
   const userSignIn = () => {
     const provider = new GoogleAuthProvider();
     return signInWithPopup(auth, provider);
   };
   return (
-    <Button color="primary" size="sm" onClick={userSignIn}>
+    <Button className="p-5" color="default" size="sm" onClick={userSignIn}>
+      <img src={Google} alt="" style={{ width: '20px', height: '20px' }} />
```

Comment:
it is good practice to add alt to something like 'google logo'. alt is used when a website fail to load the image, it shows the alt text instead.

Reply:
good tip. added